interface/weno/admin.php

```diff
@@ -50,11 +57,12 @@
     print xlt("Weno Service is Enabled")."<br><br>";
 }
 
-$drugData = $tables->drugTableInfo();
-if (!$drugData['ndc']) {
-    echo "<a href='drugPaidInsert.php?csrf_token_form=" . attr_url(collectCsrfToken()) . "' class='btn btn-default'>".xlt("Import Formularies")."</a> <br>".xlt("Be patient, this can take a while.");
+if ($tableDoesExist == true && $tableHasData == true) {
+    print xlt("Formularies are inserted into table")."<br>";
+} elseif ($tableDoesExist == false) {
+    //table needs to be installed
 } else {
-    print xlt("Formularies inserted into table")."<br>";
+    echo "<a href='drugDataInsert.php?csrf_token_form=" . attr_url(collectCsrfToken()) . "' class='btn btn-default'>".xlt("Import Formularies")."</a> <br>".xlt("Be patient, this can take a while.");
```

Comment:
isn't the table always going to exist?
To see if there is data in the tables, would just following use query (there is no need to require sql_upgrade_fx.php or use those upgrade functions in this script):
```
SELECT count(drug_id) FROM erx_weno_drugs
```

Reply:
You are right. I will change this.